hexrd/ui/calibration/polar_plot.py

```diff
@@ -151,3 +152,21 @@ def write_image(self, filename='polar_image.npz'):
             with h5py.File(filename, 'w') as f:
                 for key, value in data.items():
                     f.create_dataset(key, data=value)
+
+    def write_maud(self, filename='polar_to_maud.esg'):
+        filename = Path(filename)
+
+        with open(filename, 'wb') as fid:
+            eta_vec = np.degrees(self.angular_grid[0])
+            for i, eta in enumerate(np.average(eta_vec, axis=1).flatten()):
+                if i == 0:
+                    hstr = header0 % (i, np.linalg.norm(self.pv.tvec_s), eta)
+                else:
+                    hstr = header % (i, eta)
```

Comment:
This is a very nice design!

Reply:
This was @joelvbernier's design :slightly_smiling_face: He had a great start on getting the export correct!